### ISSUE TITLE
[BUG] Fresh installation is not working as expected when uploading documents

### ISSUE THREAD

**Issue by rSammy** [paperless-ngx/paperless-ngx#8361]
### Description

Hello,

I'm a new user to Paperless-NGX, I searched around the web then found this project. 

Being OpenSource and actually self hosted lead me to try it out before I started to stumble on so many rocks I'm on the verge of moving to Mayan EDMS or OpenKM.

First of all, the docs aren't up to date. I first wanted to install it on a bare installation on Debian before I started to realize the docs weren't up to date and that I had to read carefully what I was reading. Python dependencies wouldn't even install correctly so I had to install them one after another because it constantly broke when using requirements.txt.

I gave up and installed paperless-ngx with the docker script. I didn't use it at first because I didn't see the point of installating a container on a virtual machine. 

Everything started to miraculously work like magic. Before moving forward with customization I decided to try it out.
And it constantly breaks for whatsoever reason.

The only thing I did was trying to import a PDF file. The nightmare of debugging started :

I get the following errors :
![image](https://github.com/user-attachments/assets/999fbad0-8b82-49ce-a49a-4ffd03ea875d)

By the way my Virtual machine run on Hyper-V, dynamic memory. I don't even understand how this app can go from using 2048 MB to 20GB in seconds for whatsoever reason.

I tried using the consume folder and uploading directly from the browser, seems like there is no use.
Is there anything I can do ?

For a first time, this software looks pretty unreliable

### Steps to reproduce

Just install a fresh docker installation using the script

### Webserver logs

```bash
webserver-1  | [2024-11-26 22:25:40,513] [INFO] [paperless.tasks] WorkflowTriggerPlugin completed with:
webserver-1  | [2024-11-26 22:25:41,159] [INFO] [paperless.consumer] Consuming BL20194478.pdf
webserver-1  | [2024-11-26 22:25:41,565] [INFO] [paperless.parsing.tesseract] pdftotext exited 0
db-1         | 2024-11-27  2:25:58 23 [Warning] Aborted connection 23 to db: 'paperless' user: 'paperless' host: '172.18.0.6' (Got an error reading communication packets)
webserver-1  | [2024-11-26 22:25:58,272: ERROR/MainProcess] Process 'ForkPoolWorker-2' pid:284 exited with 'signal 9 (SIGKILL)'
webserver-1  | [2024-11-26 22:26:00,010] [ERROR] [celery.worker.request] Task handler raised error: WorkerLostError('Worker exited prematurely: signal 9 (SIGKILL) Job: 1.')
webserver-1  | Traceback (most recent call last):
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
webserver-1  |     raise WorkerLostError(
webserver-1  | billiard.einfo.ExceptionWithTraceback:
webserver-1  | """
webserver-1  | Traceback (most recent call last):
webserver-1  |   File "/usr/local/lib/python3.12/site-packages/billiard/pool.py", line 1265, in mark_as_worker_lost
webserver-1  |     raise WorkerLostError(
webserver-1  | billiard.exceptions.WorkerLostError: Worker exited prematurely: signal 9 (SIGKILL) Job: 1.
webserver-1  | """
webserver-1  | 2024-11-27 02:26:00,028 INFO reaped unknown pid 390 (terminated by SIGPIPE)
webserver-1  | [2024-11-26 22:30:00,362] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
webserver-1  | [2024-11-26 22:30:00,808] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[1e721c30-224b-49ec-84e1-49aadadabd2f] received
webserver-1  | [2024-11-26 22:30:01,882] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[1e721c30-224b-49ec-84e1-49aadadabd2f] succeeded in 0.9168302559992298s: 'No new documents were added.'
webserver-1  | [2024-11-26 22:40:00,087] [INFO] [celery.beat] Scheduler: Sending due task Check all e-mail accounts (paperless_mail.tasks.process_mail_accounts)
webserver-1  | [2024-11-26 22:40:00,441] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[eb68482e-5235-400a-87d2-ca9729d14f9c] received
webserver-1  | [2024-11-26 22:40:01,262] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[eb68482e-5235-400a-87d2-ca9729d14f9c] succeeded in 0.6296831840008963s: 'No new documents were added.'
```

### Paperless-ngx version

Latest

### Host OS

Debian 12

### Installation method

Docker - official image

### Please confirm the following

- [X] I believe this issue is a bug that affects all users of Paperless-ngx, not something specific to my installation.
- [X] This issue is not about the OCR or archive creation of a specific file(s). Otherwise, please see above regarding OCR tools.
- [X] I have already searched for relevant existing issues and discussions before opening this report.
- [X] I have updated the title field above with a concise description.

**Comment by shamoon** [comment]
Thanks for your unsolicited opinion but I would say an out of memory error is not a bug on our end.

**Comment by shamoon** [comment]
And a word of advice, next time you want help from people who make free open source software for you to use, maybe drop the snark

**Comment by rSammy** [comment]
Thank you for not helping me to resolve this problem on my fresh install on a fresh distribution VM.

Was interested because I saw it on Nextcloud. I stumbled accross some topics RAM related before eventually posting here that were not resolved either

I tried these last days finding a solution editing docker settings, however it's no use, I'll be happy to help once this project work at least without having to spend time making basics features work or having people explain where is the memory leak coming from on a fresh install. Maybe too many dependencies.

Best regards
